bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZHhad_ALP_chiral/ZHhad_ALP_chiral_proc_card.dat~

```diff
@@ -0,0 +1,20 @@
+import model ALP_chiral_UFO
+define q = u c d s b u~ c~ d~ s~ b~
+define lep+ = e+ mu+ ta+
+define lep- = e- mu- ta-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+
+define vw = w+ w-
+
+
+
+
+generate g g > z h, z > q q , h > vw  > lep+ vl lep- vl~
+add process g g > z h, z > q q , h > a  lep+ lep-
+add process g g > z h, z > q q , h > lep+ lep-  lep+ lep-
+add process g g > z h, z > q q , h > z > lep+ lep- vl vl~
+add process g g > z h, z > q q , h > z > lep+ lep- q q
+add process g g > z h, z > q q , h >  q q
+output ZHhad_ALP_chiral -nojpeg 
+
```

Comment:
this card can be removed

Reply:
thanks and sorry about that and the  duplicate cards. Should be fine now.

Reply:
Hi @menglu21, sorry I forgot to update it here. But while testing the gridpacks I realized the fix to overcome the confusion b/w two MEs does not work. Madevent gives a zero cross section and hence no events are saved post the hadronization step. That said, I'm gonna use MG to generate the process and then decay higgs with pythia8. I have not tried it yet though and if you have any suggestions please let me know. But for now, I guess we should not push cards to the repo.